clouddriver-ecs/src/integration/resources/testoperations/createServerGroup-inputs-ec2-serviceDiscovery.json

```diff
@@ -0,0 +1,40 @@
+{
+  "account": "ecs-account",
+  "application": "ecs",
+  "availabilityZones": {
+    "us-west-1": [
+      "us-west-1a",
+      "us-west-1c"
+    ]
+  },
+  "capacity": {
+    "desired": 1,
+    "max": 1,
+    "min": 1
+  },
+  "cloudProvider": "ecs",
+  "computeUnits": 256,
+  "containerPort": 80,
+  "credentials": "ecs-account",
+  "dockerImageAddress": "nginx",
+  "ecsClusterName": "integInputsEc2LegacyTargetGroup-cluster",
+  "launchType": "EC2",
+  "networkMode": "bridge",
+  "placementStrategySequence": [],
+  "reservedMemory": 512,
+  "stack": "integInputsEc2LegacyTargetGroupWithServiceDiscovery",
+  "targetGroup": "integInputsEc2LegacyTargetGroup-targetGroup",
```

Comment:
since we're not asserting on the load balancing and AFAIK it's more typical to use SD w/o it, let's remove this.

Reply:
Resolved.